server/resources/migrations/26_cli_auth.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE instant_cli_auth (
+  id uuid PRIMARY KEY,
+  secret uuid NOT NULL,
```

Comment:
The database itself should probably store a hash of the `secret` to prevent timing attacks. Not super likely here, since you'd have to have the ticket id too, but good to keep in the habit.

Example for how we store the oauth codes (the column in the table is bytea instead of uuid): https://github.com/instantdb/instant/blob/main/server/src/instant/model/instant_oauth_code.clj#L16

Reply:
Thanks, PTAL!